src/Go/boardState/boardState.ts

```diff
@@ -124,6 +131,13 @@ export function applyHandicap(board: Board, handicap: number): void {
   const handicapMoveOptions = getExpansionMoveArray(board, availableMoves);
   const handicapMoves: Move[] = [];
 
+  // Special handling for 5x5: always have handicap in the center of the board
```

Comment:
Per the bug thread, I'm not sure this is a good idea. Although I'm not sure it's a bad idea, either.

Reply:
I adjusted this to favor the tengen point as a starting position more so than other options, but it won't usually pick it